persist/src/main/resources/db/migration/V3__StewardshipAndFavoriting.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS stewardship (
+    id          SERIAL      NOT NULL PRIMARY KEY,
+    user_id     INT         NOT NULL,
+    site_id     INT         NOT NULL,
+    date_time   TIMESTAMP   DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Kind of weird this is date_time when the request calls it date. Would also be good to have this be more descriptive of when this time is from. Is it when the record was created or when the action was performed?

Reply:
Changed to performed_on and switched to date type